repl/index.html

```diff
@@ -82,7 +82,7 @@
 
         ramdaRepl(document.querySelector('.ramda-repl-target'), {
 
-          apiUrl: 'https://www.googleapis.com/urlshortener/v1/url?key=AIzaSyDhbAvT5JqkxFPkoeezJp19-S_mAJudxyk',
+          apiUrl: 'https://tinyurl.com/api-create.php',
```

Comment:
In retrospective, this is probably too much details and too flexible. It doesn't make much sense to support customization at this level. The repl should probably hide this details.

Reply:
I've never really spent any time on the REPL code.  Why do you suggest that?

Reply:
Just feel that it is unnecessary details that the caller here needs to know about. I understand there are cases where we want to customize the api key, if one is needed.

Anyway, since this code is going to last us for a relatively long period of time, we don't have to worry too much about it for now.